MAID-2237 - Updated web hosting manager UI

shankarsengalani opened the pull request:
Updated web hosting manager UI and to use latest deps.

Commit:
feat/new_ui: update hosting app to new ui

Updated web hosting app to new UI

maidsafe-highfive commented:
r? @krishnaIndia

(maidsafe_highfive has picked a reviewer for you, use r? to override)

Commit:
Handled network state on UI.

Commit:
fix/menu: update menu items

updated menu items
remove changeling.md
remove mockers api

Commit:
feat/ui: implement minor UI features

download file in chunk
reconnect UI implemented
File manager navigation implemented.

Commit:
feat/remap: implement remap feature

Implemented remap feature.

Commit:
fix/container_fetch: resolve container fetch

Resolved logic of container fetch

Commit:
disable devtool on production env

Commit:
Update Hash history on production mode
